hugolib/site.go

```diff
@@ -105,7 +106,8 @@ type SiteInfo struct {
 	Authors               AuthorList
 	Social                SiteSocial
 	Sections              Taxonomy
-	Pages                 *Pages
+	Pages                 *Pages // Includes only pages in this language
+	TranslatedPages       *Pages // Includes other translated pages, excluding those in this language.
```

Comment:
This was geared towards simplifying the code in templates to show links to other languages.. which I assume is going to be used this way most of the time.

It could be `.Translations` and we'd filter out.. 

There are also considerations about the way we structure a site.  There are those paradigms where each "default" language page has an equivalent, and in the absence of a translated page, the "default language" version is shown.

There is also a model where you can have a page in a translated language without one in the default language.  That would create a different navigation tree. In that model, you wouldn't want a fallback to a default language, because you'd be okay with having no translation for a given page.

So those models are not directly related to this comment, but they need to be kept in mind, so we provide the right data for those who want to build each models.  Now I think this would require a bit more reflection.